src/tests/jobtest.c

```diff
@@ -221,6 +239,18 @@ main(int argc, char **argv)
 				       " %" PRIu64 " rc: %d\n",
 				       pool_handles[i][j].cookie, rc);
 			}
+
+			uuid_unparse(c_uuid, cstr);
+			printf("Opening container %s\n",cstr);
```

Comment:
(style)  space required after that ',' (ctx:VxV)

Reply:
fixed